packages/f-checkout/src/components/Address.vue

```diff
@@ -115,13 +119,36 @@ export default {
     },
 
     methods: {
+        ...mapActions('checkout', [
+            'updateFulfilmentAddress'
+        ]),
         /*
         * Returns true if `field` has been touched and if it is still empty
         * The $dirty boolean changes to true when the user has focused/lost
         * focus on the input field.
         */
         isFieldEmpty (field) {
             return this.$v.addressValidations[field].$dirty && !this.$v.addressValidations[field].required;
+        },
+        updateAddressLine1(value) {
+            this.$store.dispatch('checkout/updateFulfilmentAddress', {
```

Comment:
You should be able to use the mapped action here and below?

```suggestion
            this.updateFulfilmentAddress({
```